src/lib/datepicker/calendar.html

```diff
@@ -0,0 +1,39 @@
+<div class="mat-calendar-header">
+  <button class="mat-calendar-button mat-calendar-period-button" (click)="_currentPeriodClicked()">
+    {{_label}}
+    <div class="mat-calendar-arrow"></div>
+  </button>
+  <div class="mat-calendar-spacer"></div>
+  <button class="mat-calendar-button mat-calendar-previous-button" (click)="_previousClicked()">
+    <svg xmlns="http://www.w3.org/2000/svg" width="24px" height="24px" viewBox="0 0 24 24"
+         fill="#000000">
```

Comment:
These are "keyboard arrow left" and "keyboard arrow right" from go/icons. I'd rather not mess with the contents of the svgs. I'm embedding them because its a pattern we already follow elsewhere in the code (I assume to avoid incurring additional resource requests). see: https://github.com/angular/material2/blob/master/src/lib/checkbox/checkbox.html#L24